apps/backend/src/services/accounts.js

```diff
@@ -176,137 +172,141 @@ async function hydrateDemoData({ userId }) {
         ],
       },
     },
-    false,
-    {
+  },
+  {
+    title: 'Demoquestion FT',
+    type: 'FREE',
+    content: 'Describe a main principle of a social market economy.',
+    options: false,
+  },
+  {
+    title: 'Demoquestion NR',
+    type: 'FREE_RANGE',
+    content: 'Estimate the length of the **longest** river in the world (answer in kilometres).',
+    options: {
       FREE_RANGE: {
-        randomized: false,
         restrictions: { min: 0, max: 8000 },
-        choices: [],
       },
     },
-  ]
-  const demoQuestions = []
+  },
+]
+
+function prepareDemoSessionData({ id, name, user, blockInstances, blockStatus = 'PLANNED', ...rest }) {
```

Comment:
...rest allows us to catch any arguments that are not specifically named, so we can override arbitrary values of the object by using ...rest in the final line below